src/scripting/lua_map.cc

```diff
@@ -1944,6 +1974,19 @@ int LuaTribeDescription::get_name(lua_State* L) {
 	return 1;
 }
 
+/* RST
+   .. attribute:: directory
+
+         (RO) a :class:`string` with the path of the tribe's initialization scripts
+*/
+
+int LuaTribeDescription::get_directory(lua_State* L) {
```

Comment:
I cannot find a usage of this function either. Are they intended for add-on developers?

Reply:
`data/tribes/scripting/help/tips.lua` line 34, to locate the tribe's game tips script without hardcoding the paths